src/replicache.ts

```diff
@@ -277,6 +277,16 @@ export class Replicache<MD extends MutatorDefs = {}> {
    */
   onSync: ((syncing: boolean) => void) | null = null;
 
+  /**
+   * `onClientMissing` is called when the persistent client has been garbage
+   * collected. This can happen if the client has not been used for over a week.
+   *
+   * The default behavior is to reload the page (using `location.reload()`). Set
+   * this to `null` or provide your own function to prevent the page from
+   * reloading automatically.
+   */
+  onClientMissing: (() => void) | null = reload;
```

Comment:
Maybe we should only assign the `reload` in the case we're in a document so at least workers get an error?

Reply:
How about `onClientMissingError`.

Reply:
Perhaps `onClientStateNotFound`.

Reply:
> Maybe we should only assign the reload in the case we're in a document so at least workers get an error?

reload is only done if there is a `location` global. We could do the check in the constructor but this seems simpler.

Reply:
Not sure if we're saying the same thing. What I'm saying is that it's desirable that if a worker encounters a `ClientNotFoundError` that the error actually get reported to the console, not that it's silently swallowed.